test/specs/WasteRecievingAPI/CreateWasteMovement/hazardous-properties-indicator.test.js

```diff
@@ -0,0 +1,90 @@
+import { describe, it, expect, beforeEach } from '@jest/globals'
+import { generateBaseWasteReceiptData } from '../../../support/test-data-manager.js'
+
+describe('Hazardous Properties Indicator Validation', () => {
+  let wasteReceiptData
+
+  beforeEach(() => {
+    wasteReceiptData = generateBaseWasteReceiptData()
+  })
+
+  describe('Valid Hazardous Indicator', () => {
+    it('should accept waste receipt when hazardous indicator is set to true', async () => {
+      wasteReceiptData.waste[0].hazardous = {
+        containsHazardous: true
+        // Missing components array
+      }
+
+      const response =
+        await globalThis.apis.wasteMovementExternalAPI.receiveMovement(
+          wasteReceiptData
+        )
+
+      expect(response.statusCode).toBe(200)
+      expect(response.data).toHaveProperty('globalMovementId')
+    })
+
+    it('should accept waste receipt when hazardous indicator is set to false', async () => {
+      wasteReceiptData.waste[0].hazardous = {
+        containsHazardous: false
+      }
+
+      const response =
+        await globalThis.apis.wasteMovementExternalAPI.receiveMovement(
+          wasteReceiptData
+        )
+
+      expect(response.statusCode).toBe(200)
+      expect(response.data).toHaveProperty('globalMovementId')
+    })
+
+    it('should accept waste receipt when hazardous indicator is set to true and components are provided', async () => {
```

Comment:
not sure if this is a possible scenario but should we have tests for containsHazardous: false with components provided or Invalid boolean value tests (null, undefined, strings)?